snyk/snyk-gradle-plugin
# chore: update node 8 to 16

#### What does this PR do?
In preparation for the Gradle plugin improvements this:
- updates node version from 8 to 16 😨
- updates documentation including changing formatting of the README file (result of running `npm run lint`)

#### How should this be manually tested?
1. Clone and test the following Gradle projects with snyk CLI: [vuln project 1](https://github.com/snyk-fixtures/gradle-vuln-upgrade), [vuln project 2](https://github.com/snyk-fixtures/simple-gradle-repo), [non-vuln Android project](https://github.com/snyk-fixtures/gradle-android-projects), [non-vuln project]( https://github.com/snyk-fixtures/gradle-multi-sub-projets)
2. Check the CLI returns the same results for the above projects when using this branch of snyk-gradle-plugin. To do this:
- Pull this branch of snyk-gradle-plugin 
- Clone snyk/cli
- On the master branch, in the CLI package.json change `"snyk-gradle-plugin": "3.17.0"` to  `"snyk-gradle-plugin": "../snyk-gradle-plugin"`
- Run: `npm i && npm run build`
- Use CLI locally by adding an alias snyk-local='SNYK_API="https://app.dev.snyk.io/api/v1" node /Users/XXXX/cli/dist/cli' -> pointing to your local CLI repo
- Run `snyk-local test` with: [vuln project 1](https://github.com/snyk-fixtures/gradle-vuln-upgrade), [vuln project 2](https://github.com/snyk-fixtures/simple-gradle-repo), [non-vuln Android project](https://github.com/snyk-fixtures/gradle-android-projects), [non-vuln project]( https://github.com/snyk-fixtures/gradle-multi-sub-projets)

#### What are the relevant tickets?
[TARDIS-830](https://snyksec.atlassian.net/browse/TARDIS-830)
